README.md

```diff
@@ -147,14 +147,23 @@ linked and embedded:
 make
 ```
 
-The "package maintainer" way, without any embedded binaries (requires that the
-required binaries are provided separately at runtime):
+The "package maintainer" way, without building and embedding the required
+binaries. This assumes necessary binaries are provided separately at runtime:
 
 ```shell
 make EMBEDDED_BINS_BUILDMODE=none
 ```
 
-The embedded binaries can be built on their own:
+As mentioned previously, Docker build integration is enabled by default.
+Building k0s without Docker using the system toolchain can be done as follows:
```

Comment:
What do you mean by "system toolchain", maybe give an example what this means in this context

Reply:
PTAL. I tried to clarify this sentence.